src/Parser/parser.js

```diff
@@ -0,0 +1,24 @@
+import parseNormalInput from './SubParser/parseNormalInput.js';
+import parseCustomInput from './SubParser/parseCustomInput.js';
+
+import isFirstCharNumber from '../Util/isFirstCharNumber.js';
+import throwError from '../Error/handleError.js';
+import { ERROR_PREFIX, ERROR_MESSAGES } from '../Constraints/Constraints.js';
+// 에러를 처리하는 함수
+
+// 에러를 출력하는 함수
+export default function parseString(str) {
+  if (!str && str === '') {
+    return 0;
+  }
+  if (str === null || str === undefined) {
+    throwError(`${ERROR_PREFIX}${ERROR_MESSAGES.EMPTY_STRING}`);
```

Comment:
이부분에서 템플릿 리터럴을 사용하신 이유가 있으신가요?
```suggestion
    throwError(ERROR_PREFIX + ERROR_MESSAGES.EMPTY_STRING);
```

Reply:
템플릿 리터럴이 손에 익어서 그렇습니다. 지금 보니 저렇게 스트링을 합치는게 더 맞는 것 같네요. 피드백 감사합니다!